lib/common/template.js

```diff
@@ -41,7 +41,7 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
             Constants.Templates.Directory
         ).then(function (templates) {
             var promises = _.map(templates, function (contents, name) {
-                return self.put(name, contents);
+                return self.put(name, contents.toString());
```

Comment:
Only concern here is that this will break if context is null or undefined. I'm assuming that is not possible here.